sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -525,7 +525,7 @@ object Radar {
         if (fraud) {
             RadarSettings.setSharing(this.context, false)
         }
-        application?.registerActivityLifecycleCallbacks(RadarActivityLifecycleCallbacks(fraud))
+        application?.registerActivityLifecycleCallbacks(RadarActivityLifecycleCallbacks(this.logger, fraud))
```

Comment:
Minor style nit, but we're passing `logger` instead of `this.logger` in the rest of the initialize

Reply:
Good callout, updated